react-form-controls/app-name/src/RegistrationFormControlled.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+
+export default function RegistrationFormControlled() {
+  const [userName, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(e) {
+    e.preventDefault();
+
+    const form = e.target;
+    const formData = new FormData(form);
+    const formJson = Object.fromEntries(formData.entries());
```

Comment:
For a controlled form, you don't need to use FormData since you have the values already in your state variables. Just log the state variables directly.